cfn.yaml

```diff
@@ -186,6 +186,13 @@ Resources:
               "featureId": "braze_epic",
               "platformId": "ios"
             }
+        - Arn: !GetAtt Lambda.Arn
+            Id: android-braze-epic-scheduler
```

Comment:
The indentation of `Id` and `Input` do not match the ones above. My knowledge of yaml is limited but I would expect this to not work correctly with ClourFormation. Have you deployed this to CODE environment using RiffRaff?

Reply:
Thanks.. infact it didn't work. Fixed now.